lib/operations/add_user.js

```diff
@@ -22,7 +22,11 @@ class AddUserOperation extends CommandOperation {
     const options = this.options;
 
     // Get additional values
-    let roles = Array.isArray(options.roles) ? options.roles : [];
+    let roles = Array.isArray(options.roles)
+      ? options.roles
+      : typeof options.roles === 'string'
+      ? [options.roles]
+      : [];
```

Comment:
This was a bug I picked up by using the TS interface as a guide, this seems like it was / is the intention, also is a bug in master (needs port)

Reply:
:+1: but I think convention has been to break out these nested ternaries into explicit if statements (I think we used to enable an eslint rule for this, and it's been lost in the shuffle?).